<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeToPixEditForm</class>
 <widget class="QWidget" name="WelcomeToPixEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="50,50">
   <item>
    <widget class="QLabel" name="pixEditSplashLogo">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>37</pointsize>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/images/PixEditLogoV1.PNG</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="loadProjectButton">
       <property name="text">
        <string>Load Project</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacerTop">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="createResolutionCBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="projectNameLineEdit">
       <property name="text">
        <string>myNewSprite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newProjectButton">
       <property name="text">
        <string>New Project</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacerBottom">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="aboutPixEditButton">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
